plugins/subnet/plugin.spec.yaml

```diff
@@ -19,53 +19,58 @@ tags:
 - utilities
 hub_tags:
   use_cases: [data_utility, threat_detection_and_response]
-  keywords: [subnet, ip, netmask, network, utilities]
+  keywords: [subnet, ip_address]
   features: []
 actions:
   calculate:
     title: Calculate
-    description: Returns Subnet Information for IP and Netmask
+    description: Return subnet information for IP and Netmask
     input:
       cidr:
         title: Network
-        description: Network in CIDR Notation, E.g. 192.168.1.1/24
+        description: Network in CIDR notation, E.g. 198.51.100.0/24
         type: string
         required: true
+        example: 198.51.100.0/24
     output:
       ip:
-        title: IP Address
+        title: IP
```

Comment:
```suggestion
        title: IP Address
```

Reply:
@mrinehart-r7 @jrose-r7 I changed these titles to the same as the names(identifiers) of the outputs and due to the fact that according to the documentation, the description should not be the same as the title. Of course I can change it if it is necessary.

Reply:
@dsliwinski-r7 the description can be the same as the title - no issues with that!